core-bundle/src/EventListener/DataContainer/PageUrlListener.php

```diff
@@ -203,7 +215,7 @@ private function recursiveValidatePages(int $pid, PageModel $rootPage): void
 
         /** @var PageModel $page */
         foreach ($pages as $page) {
-            if ($page->alias) {
+            if ($page->alias && $this->pageRegistry->isRoutable($page)) {
```

Comment:
This condition is also checked at the beginning of the `aliasExists()` method. Are both checks necessary?

Reply:
yes, because this checks `$page` but we forward `$rootPage` to the `checkAlias` method

Reply:
But I have improved that in ed539ca88ea7797492e8077dca7f0f6666a05e6c